### PR TITLE
SR-029 Small bug fixes for the first Release [DK,PO]

### PR DESCRIPTION
**Pull request by astrokit**

### CODE REVIEW

File: app/src/main/res/layout/fragment_route_list.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <ListView
         android:id="@+id/list"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
```
Comment (marha333) [inline app/src/main/res/layout/fragment_route_list.xml]: We could fix this in activity_main.xml (this way it works for all fragments that are displayed)

You could try wrapping the Fragment and the BottomNavigationView with a relative Layout. Feel free to ask if you need any help ;)

**Review by marha333 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_route_list.xml (hunk 1 above)
Comment (jerlacher) [reply app/src/main/res/layout/fragment_route_list.xml]: agree :)

**Review by jerlacher — APPROVED** [review]
Would implement Haradl's change.
Otherwise everything looks great 👍

**Review by marha333 — APPROVED** [review]
Great :)

**Pull request merged**